workers/generic-worker/main.go

```diff
@@ -623,6 +626,12 @@ func (task *TaskRun) validatePayload() *CommandExecutionError {
 		panic(err)
 	}
 	if _, exists := payload["image"]; exists {
+		if !config.EnableD2G {
+			return MalformedPayloadError(errors.New(`docker worker payload detected, but D2G is not enabled on this worker pool.
+If you need D2G to translate your Docker Worker payload so Generic Worker can process it, please do one of two things:
+	1. Contact the owner of the worker pool and ask for D2G to be enabled.
```

Comment:
It would be really neat here to provide a link to the Worker Pool definition in the UI, since we have the root URL and can create it. This way, they can see who the owner is directly if they follow the link.